docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-potential-process-injection-via-powershell.asciidoc

```diff
@@ -0,0 +1,141 @@
+[[prebuilt-rule-0-16-2-potential-process-injection-via-powershell]]
+=== Potential Process Injection via PowerShell
+
+Detects the use of Windows API functions that are commonly abused by malware and security tools to load malicious code or inject it into remote processes.
```

Comment:
Flagging this because I couldn't identify what "it" (at the end of the sentence) was referencing.